core/src/main/scala/org/apache/spark/storage/ExternalBlockStore.scala

```diff
@@ -107,13 +137,19 @@ private[spark] class ExternalBlockStore(blockManager: BlockManager, executorId:
       externalBlockManager.map(_.removeBlock(blockId)).getOrElse(true)
     } catch {
       case NonFatal(t) =>
-        logError(s"error in removing $blockId", t)
+        logError(s"Error in removing $blockId", t)
```

Comment:
anyway please just change the logging messages to make them very explicit that they are function calls. We should either do this, or make them English sentences.